packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## 0.1.2
 
 * Fix variouse bugs.
+* Update webview_flutter to 3.0.4.
+* Update webview_flutter_tizen to 0.5.1.
+* Update dependency package.
```

Comment:
Thanks for the changes, I'm not sure if there are other meanings to "Update dependency package" other than bumping up `webview_flutter` and `webview_flutter_tizen`. This line is not needed if you meant those two packages.

Reply:
Ah~ As I mentioned above, actually @bbrto21 's patch was 0.1.2 and it will be released with this PR. So I thought I needed `Update dependency package` about @bbrto21 patch. What do you think? :)

Reply:
@seungsoo47 I think you don't need a last line. 
```suggestion
```